Assets/Scripts/Models/Events/HeadlineGenerator.cs

```diff
@@ -40,7 +33,7 @@ public void AddHeadline(string headline, bool displayImmediately = true, bool ke
     {
         if (keepInQueue)
         {
-            headlines.Add(headline);
+            PrototypeManager.Headline.Add(new Headline(headline));
```

Comment:
The only place we do it for now is in the ScheduleEvents and I didn't like that already :p

I won't block this PR for that but, we should probably open an issue to discuss about it. Just defining the prototypes would be enough for me.

I wanted to do the same for the trading request but restrained myself until now.